src/Compilers/Core/Portable/InternalUtilities/FatalError.cs

```diff
@@ -67,6 +67,14 @@ public static void OverwriteHandler(Action<Exception> value)
             s_fatalHandler = value;
         }
 
+        // Same as setting the Handler property except that it avoids the assert.  This is useful in 
+        // test code which needs to verify the handler is called in specific cases and will continually
+        // overwrite this value.
+        public static void OverwriteNonFatalHandler(Action<Exception> value)
+        {
+            s_fatalHandler = value;
```

Comment:
How is this different than the method above?

Reply:
It is not. However, if you asked how it's *supposed* to be different from the method above, the answer is totally different. 😂

Reply:
This is now fixed. I amended the previous commit since the change was so small.